gettyimagesapi-sdk/src/main/java/com/gettyimages/api/HttpClientErrorException.java

```diff
@@ -0,0 +1,23 @@
+package com.gettyimages.api;
+
+import org.apache.http.StatusLine;
+
+public class HttpClientErrorException extends SdkException {
```

Comment:
Nice! 👍 on the `HttpClientErrorException` / `HttpSystemErrorException` differentiation thing.